static/styles/banners.css

```diff
@@ -0,0 +1,65 @@
+body{
+    margin:0px;
+    /* font-family: Dosis; */
+}
+@media (min-width: 1025px) {
+  body{
+    max-width: 80%;
```

Comment:
dont try to directly update style of the body tag. Add a container div under body and update its styles if needed